bot/config.py

```diff
@@ -0,0 +1,15 @@
+import dataclasses
+import typing
+
+import dotenv
+
+
+@dataclasses.dataclass
+class Config:
+    TOKEN: str
+    PISTON_URL: str
+    GODBOLT_URL: str
+
+
+env_vars: typing.Any = dotenv.dotenv_values()
```

Comment:
whoah how did I not know about this

also can we use `dict[str, str]` and just cast the result if needed?

Reply:
Done